content/4_Gold/Digit_DP.problems.json

```diff
@@ -0,0 +1,46 @@
+{
+  "MODULE_ID": "digit-dp",
+  "focus": [
+    {
+      "uniqueId": "usaco-435",
+      "name": "Odometer",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=435",
+      "source": "Old Silver",
+      "difficulty": "Easy",
+      "isStarred": false,
+      "tags": ["DP"],
+      "solutionMetadata": {
+        "kind": "USACO",
+        "usacoId": "435"
+      }
+    }
+  ],
+  "problems": [
+    {
+      "uniqueId": "usaco-1307",
+      "name": "Piling Papers",
+      "url": "http://usaco.org/index.php?page=viewproblem2&cpid=1307",
+      "source": "Gold",
+      "difficulty": "Hard",
+      "isStarred": false,
+      "tags": ["DP"],
+      "solutionMetadata": {
+        "kind": "USACO",
+        "usacoId": "1307"
+      }
+    },
+    {
+      "uniqueId": "usaco-1115",
+      "name": "Count the Cows",
+      "url": "http://usaco.org/index.php?page=viewproblem2&cpid=1115",
+      "source": "Gold",
+      "difficulty": "Very Hard",
```

Comment:
I don't get why this is "Very Hard" and the first problem is "Easy" - these problems aren't that different in difficulty IMO. maybe "Hard" for this one and "Normal" for the first one?